rostran/cli/__main__.py

```diff
@@ -102,18 +117,23 @@ def transform(
             target_format = TargetTemplateFormat.Yaml
         else:
             target_path = "template.json"
+        if source_format == SourceTemplateFormat.ROS:
+            target_path = 'template'
     elif target_format == TargetTemplateFormat.Auto:
         if target_path.endswith((".yaml", ".yml")):
             target_format = TargetTemplateFormat.Yaml
         elif target_path.endswith(".json"):
             target_format = TargetTemplateFormat.Json
         else:
-            raise exceptions.TemplateNotSupport(path=target_path)
+            if source_format != SourceTemplateFormat.ROS:
+                raise exceptions.TemplateNotSupport(path=target_path)
 
     target_path = os.path.abspath(target_path)
     path = Path(target_path)
     if path.exists():
-        raise exceptions.TemplateAlreadyExist(path=target_path)
+        if not force:
```

Comment:
`if path.exists() and not force` would be better

Reply:
👌🏻